apps/dashboard/src/app/(dashboard)/dashboard/connect/ecosystem/[ecosystemId]/components/server/partners-table.tsx

```diff
@@ -8,7 +9,18 @@ import { useDeletePartner } from "../../hooks/use-delete-partner";
 import { usePartners } from "../../hooks/use-partners";
 
 export function PartnersTable({ ecosystem }: { ecosystem: Ecosystem }) {
-  const { partners } = usePartners({ ecosystemId: ecosystem.id });
+  const { partners, isLoading } = usePartners({ ecosystemId: ecosystem.id });
+
+  if (isLoading) {
+    return (
+      <div className="space-y-2">
+        {Array.from({ length: 5 }).map((_, i) => (
+          // biome-ignore lint/suspicious/noArrayIndexKey: false
```

Comment:
better explanation here please


```suggestion
          // biome-ignore lint/suspicious/noArrayIndexKey: static list with index as key
```